tools/StaticAnalysis/Exceptions/HelpIssues.csv

```diff
@@ -142,3 +142,9 @@
 "Microsoft.AzureStack.Commands.dll","Microsoft.AzureStack.Commands.dll-help.xml","Microsoft.AzureStack.Commands.SetSubscription","1","6050","Help missing for cmdlet Set-AzsSubscription implemented by class Microsoft.AzureStack.Commands.SetSubscription","Add Help record for cmdlet Set-AzsSubscription to help file."
 "D:\workspace\powershell-sign\src\Package\Release\ResourceManager\AzureResourceManager\AzureRM.AzureStackStorage\Microsoft.Azure.Commands.Resources.dll","D:\workspace\powershell-sign\src\Package\Release\ResourceManager\AzureResourceManager\AzureRM.AzureStackStorage\Microsoft.Azure.Commands.Resources.dll-Help.xml","D:\workspace\powershell-sign\src\Package\Release\ResourceManager\AzureResourceManager\AzureRM.AzureStackStorage\Microsoft.Azure.Commands.Resources.dll","0","6000","D:\workspace\powershell-sign\src\Package\Release\ResourceManager\AzureResourceManager\AzureRM.AzureStackStorage\Microsoft.Azure.Commands.Resources.dll has no matching help file","Make sure a dll Help file for D:\workspace\powershell-sign\src\Package\Release\ResourceManager\AzureResourceManager\AzureRM.AzureStackStorage\Microsoft.Azure.Commands.Resources.dll exists and it is being copied to the output directory."
 "Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.dll-Help.xml","Microsoft.Azure.Commands.EventHub.Commands.Namespace.SetAzureEventHubNamespace","1","6050","Help missing for cmdlet Set-AzureRmEventHubNamespace implemented by class Microsoft.Azure.Commands.EventHub.Commands.Namespace.SetAzureEventHubNamespace","Add Help record for cmdlet Set-AzureRmEventHubNamespace to help file."
+"Microsoft.Azure.Commands.IotHub.dll","Microsoft.Azure.Commands.IotHub.dll-help.xml","Microsoft.Azure.Commands.Management.IotHub.AddAzureRmIotHubCertificate","1","6050","Help missing for cmdlet Add-AzureRmIotHubCertificate implemented by class Microsoft.Azure.Commands.Management.IotHub.AddAzureRmIotHubCertificate","Add Help record for cmdlet Add-AzureRmIotHubCertificate to help file."
```

Comment:
Why are you suppressing these messages? They should not be an issue now that you have generated help files.

Reply:
reverted